### DISCUSSION TITLE
The FTMO Challenge

### DISCUSSION THREAD

**Discussion by kenorb** [EA31337/EA31337#367] [Ideas]
Refs:

- https://ftmo.com/en/faq/
- https://ftmo.com/en/faq/step-1-ftmo-challenge/

**Comment by Mugen8Sekai** [comment]
Hi, nice to meet you. I've been trying your bot for a few weeks now, but I'm not getting good results yet. Any tips for this challenge? it has some restrictions that make it even more complicated. Thanks.

**Reply by kenorb** [reply]
Can you list all trading rules/limitations that you're looking for to be implemented?
Which version are you using?

**Comment by Mugen8Sekai** [comment]
My first step is to complete the challenge with no event or news restrictions. The main problem is the daily loss limit of €500 and the accumulated loss limit of €1000. With 10000€ and a strong leverage it is easy to exceed 500€ euros per day, just like in the tests to chain several days of losses.
As for the future, for the limit of trading during the news I have thought of stopping the algorithmic trading manually and use the parameters of schedules that are already available in the bot to not operate in the closing hours (overnight). Other ideas?

Currently I have used EA31337 Lite, Advanced and Rider. I have compiled the different versions with MetaEditor 5, changing to have less inputs or more inputs. Some simulations have taken more than 48h. I am focused on M1 and M5 strategies (may be better strategies in other intervals?).

With a good strategy I think you can avoid losing the accumulated 1000€ (that the account stays above 9000€), but it is useless to have accumulated 13000€ if one bad day you lose 500€ and lose the funded account.

Thanks!

**Reply by kenorb** [reply]
* daily loss limit of €500

  It's not implemented, I've created related ticket to track on this at <https://github.com/EA31337/EA31337-strategies-meta/issues/74>. It's not easy to implement, as the price always can move suddenly. So this can be either done by shortening trailing stops so total loss is 500, or by trading equity and closing all orders when <500 is reached. But this cannot be for sure, because when you've 400 equity on minus, any sudden move can go beyond 500 and EA could not react on time.

* accumulated loss limit of €1000

  If you accumulate €1000 losses in total, then you won't be able to trade further, so this can be only achieved by lowering the risk or by optimizing EA more for the given market conditions.

* the limit of trading during the news

  This is going to be implemented in the next EA version as part of [Meta News](https://github.com/EA31337/Strategy-Meta_News) strategy where you can choose to not trade or trade with different strategy during the news events.

* to not operate in the closing hours (overnight)

  To not trade overnight, you can use Signal time filter. See: <https://github.com/EA31337/EA31337/wiki/Input-parameters>.

* I am focused on M1 and M5 strategies

  M1 and M5 are not so profitable as M15 and beyond, so try these with higher timeframes instead. So consider testing these strategies on higher time intervals to identify potentially better-performing strategies.
